config_default.ini

```diff
@@ -16,6 +16,8 @@ google_maps_api_key =
 bing_maps_api_key =
 # for 3D view, https://cesium.com/ion/
 cesium_api_key =
+# https://www.mapbox.com/account/access-tokens
+mapbox_api_access_token =
```

Comment:
Is this required?

Reply:
yes, that is necessary because map source of leaflet is from mapbox currently

Reply:
Ok, can you add a comment above to the `google_maps_api_key`, that the Google API key is not used currently.

Reply:
done!